pkg/trace/api/api.go

```diff
@@ -128,6 +128,7 @@ func (r *HTTPReceiver) Start() {
 	mux.HandleFunc("/v0.3/services", r.handleWithVersion(v03, r.handleServices))
 	mux.HandleFunc("/v0.4/traces", r.handleWithVersion(v04, r.handleTraces))
 	mux.HandleFunc("/v0.4/services", r.handleWithVersion(v04, r.handleServices))
+	mux.Handle("/profile", r.profileProxyHandler())
```

Comment:
Do we want to version this endpoint like the others?

Reply:
I have no idea. I have asked myself this question too. Some possible answers:

* Using something similar as the Go language does `/x/profile` meaning that it's an "experimental" or "beta" endpoint.
* Starting off with versions now, but if so, how would that look? We already have versions at the root. IMHO we should have had `/api/v0.1/` instead of `/v0.1/`
* Add versions later on if there will be changes.

I definitely agree that this is not the best endpoint path.

Reply:
Changed to `/proxy/profile` for now, but we should discuss.

Reply:
@knusbaum WDYT about the endpoint now?